test/mock/kernel/arch_mock.zig

```diff
@@ -6,6 +6,7 @@ const gdt = @import("gdt_mock.zig");
 const idt = @import("idt_mock.zig");
 const vmm = @import("vmm_mock.zig");
 const paging = @import("paging_mock.zig");
+const Serial = @import("../../../src/kernel/serial.zig").Serial;
```

Comment:
Why not create a serial_mock.zig file?

Reply:
The serial file is literally just the struct though so I didn't think that would be necessary xD